slides/k8s/kubectlget.md

```diff
@@ -79,26 +79,102 @@
 
 ---
 
-## What's available?
-
-- `kubectl` has pretty good introspection facilities
+## Exploring types and definitions
 
 - We can list all available resource types by running `kubectl api-resources`
   <br/>
   (In Kubernetes 1.10 and prior, this command used to be `kubectl get`)
 
-- We can view details about a resource with:
+- We can view the definition for a resource type with:
   ```bash
-  kubectl describe type/name
-  kubectl describe type name
+  kubectl explain type
   ```
 
-- We can view the definition for a resource type with:
+- We can view the definition of a field in a resource, for instance:
   ```bash
-  kubectl explain type
+  kubectl explain node.spec
+  ```
+
+- Or get the full definition of all fields and sub-fields:
+  ```bash
+  kubectl explain node --recursive
+  ```
+
+---
+
+## Introspection vs. documentation
+
+- We can access the same information by reading the [API documentation](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/)
```

Comment:
Do we want a version number in these slides? Would https://kubernetes.io/docs/reference/#api-reference maybe be better in this case?

Reply:
Question still remains - I'm worried about pointing to a specific version and then needing to update it frequently.

Reply:
You're right, it's much better to point to the link you proposed. I don't know why I had missed your suggestion, sorry!